### ISSUE TITLE
Windows binaries doesn't simply work

### ISSUE THREAD

**Issue by Darkness4** [Darkness4/fc2-live-dl-go#46]
... somehow, this issue never got reported.

Binary is still dynamically linked:

```batch
PS > dumpbin.exe /dependents .\fc2-live-dl-go-windows-amd64.exe
Microsoft (R) COFF/PE Dumper Version 14.40.33812.0
Copyright (C) Microsoft Corporation.  All rights reserved.


Dump of file .\fc2-live-dl-go-windows-amd64.exe

File Type: EXECUTABLE IMAGE

  Image has the following dependencies:

    avcodec-60.dll
    avformat-60.dll
    avutil-58.dll
    KERNEL32.dll
    msvcrt.dll

  Summary

        1000 .CRT
       68000 .bss
       53000 .data
        1000 .edata
        2000 .idata
       1C000 .pdata
      490000 .rdata
       17000 .reloc
      46E000 .text
        1000 .tls
        1000 .xdata
```

Issue appeared after migrating from MXE to Portage. [Diff](https://github.com/Darkness4/fc2-live-dl-go/compare/v1.1.3+2...v1.1.3+3)

**Comment by Darkness4** [comment]
Rollback to MXE for version v1.7.1 and v1.8.0, and maybe for all future version.

**Comment by Darkness4** [comment]
Closed with release v1.7.1 and commit 11834291978097f8cd1bb6207ec7d2206000fb68
